django/contrib/postgres/forms/array.py

```diff
@@ -178,6 +178,9 @@ def __init__(self, base_field, size, *, remove_trailing_nulls=False, **kwargs):
         kwargs.setdefault('widget', widget)
         super().__init__(**kwargs)
 
+    def to_python(self, value):
+        return self.clean(value)
```

Comment:
I'm not sure if we need an entire `clean()` logic. `value` is already a list so calling `base_field.to_python()` on items should be enough, e.g.
```python
def to_python(self, value):
    value = super().to_python(value)
    return [self.base_field.to_python(item) for item in value]
```
What do you think?